File: test/py/case50_class_instance.py
class Box50:
    def __init__(self, seed: int) -> None:
        self.seed = seed

    def next(self) -> int:
        self.seed = self.seed + 1
        return self.seed


if __name__ == "__main__":
    b: Box50 = Box50(3)
    print(b.next())

File: test/cs/case50_class_instance.cs
using System;

public static class Program
{
    public class Box50
    {
        public int seed;

        public Box50(int seed)
        {
            this.seed = seed;
        }
        public int next()
        {
            this.seed = (this.seed + 1);
            return this.seed;
        }
    }

    public static void Main(string[] args)
    {
        Box50 b = new Box50(3);
        Console.WriteLine(b.next());
    }
}
